streamer/pulp/streamer/server.py

```diff
@@ -95,16 +95,15 @@ def download_failed(self, report):
         :param report: The download report for this request.
         :type  report: nectar.report.DownloadReport
         """
-        error_report = report.error_report.copy()
-        error_report['url'] = report.url
-        logger.info(_('Download of %(url)s failed: HTTP %(response_code)s '
-                      '%(response_msg)s' % error_report))
```

Comment:
I thought about keeping it, but did some manual testing and realized that Nectar will log pretty much the same thing (https://github.com/pulp/nectar/blob/master/nectar/downloaders/threaded.py#L225-L226) if it's an HTTP status that isn't 200, and urllib3 logs connection-related problems.